docker/Dockerfile.centos.7

```diff
@@ -1,34 +1,12 @@
-# Docker build file to create an image running centos7 with MySQL and build
-# tools installed
-
-FROM centos/mysql-56-centos7:latest
+# Docker build file to create an image with centos 7
+FROM centos:7
```

Comment:
Same as previous commit, I can't see how these new files are used. I guess they aren't so far, which is weird.